test/player_test.cpp

```diff
@@ -8,13 +8,13 @@ namespace devilution {
 extern bool TestPlayerDoGotHit(int pnum);
 }
 
-int RunBlockTest(int frames, int flags)
+int RunBlockTest(int frames, ItemSpecialEffect flags)
 {
 	int pnum = 0;
 	auto &player = Players[pnum];
 
 	player._pHFrames = frames;
-	player._pIFlags = flags;
+	player._pIFlags = static_cast<ItemSpecialEffect>(flags);

```

Comment:
this cast is unnecessary now, flags is already of type `ItemSpecialEffect`
```suggestion
	player._pIFlags = flags;
```